peer/brontide.go

```diff
@@ -3009,7 +2992,26 @@ func (p *Brontide) handleLocalCloseReq(req *htlcswitch.ChanClose) {
 			return
 		}
 
-		p.queueMsg(shutdownMsg, nil)
+		link := p.fetchLinkFromKeyAndCid(chanID)
+		if link != nil {
```

Comment:
`if link == nil`

Reply:
we can't register a flush hook on a nil channel link though..